js/web/test/e2e/run.js

```diff
@@ -109,6 +109,8 @@ async function testAllBrowserCases({ hostInKarma }) {
   await runKarma({ hostInKarma, main: './browser-test-wasm-path-override-filename.js', ortMain: 'ort.wasm.min.js'});
   await runKarma({ hostInKarma, main: './browser-test-wasm-path-override-prefix.js'});
   await runKarma({ hostInKarma, main: './browser-test-wasm-path-override-prefix.js', ortMain: 'ort.wasm.min.js'});
+  await runKarma({ hostInKarma, main: './browser-test-wasm-Image-tensor-Image.js'});
+  await runKarma({ hostInKarma, main: './browser-test-wasm-Image-tensor-Image.js', ortMain: 'ort.wasm.min.js'});
```

Comment:
as we are not calling into ORT at all, we don't need this one

Reply:
OK cool I will remove it